.gitignore

```diff
@@ -29,3 +29,5 @@ Thumbs.db
 node_modules
 
 dist
+coverage
+yarn-error.log
```

Comment:
For the items that are always in the root directory, we need to begin the path with a `/`.